core/src/main/java/com/backblaze/b2/client/structures/MetadataDirective.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright 2019, Backblaze Inc. All Rights Reserved.
+ * License https://www.backblaze.com/using_b2_code.html
+ */
+
+package com.backblaze.b2.client.structures;
+
+public enum MetadataDirective {
```

Comment:
Are these ever returned by the service?  In other places, the SDK uses strings rather than enums to allow for new enum items defined by the service.

Reply:
They are not returned by the service. I can switch to strings to be safe.